### DISCUSSION TITLE
Componente de conteúdo avançado C

### DISCUSSION THREAD

**Discussion by vinrosa** [Telefonica/mistica-design#1197] [Proposals]
### Goal

Criar um componente de conteúdo ”avançado” que será utilizado no componente data card através da propriedade de slot e que possa ser reutilizável de diversas maneiras e por combinações diferentes.

### Context

Os cards apresentados no novo conceito para o novo app Vivo possuem elementos de conteúdo que não são suportados pelo atual componente data card, a favor de suportar esse conteúdo, e com um racional de criar o menor número possível de variações, concluímos que com 4 novos componente de conteúdo  para uso da Vivo atenderemos os casos atuais da aba de assinaturas.
Esse componente suporta os componentes de Assinatura isolada, Suporte e Cancelamento

### Impact

Não gera impacto em componentes já existentes na biblioteca Mística.
Componente vai ser utilizado na lib de componentes avançados

### Discussion Questions

Alguma sugestão de melhoria?

### Additional considerations

Componentes relacionados:
https://github.com/Telefonica/mistica-design/discussions/1195
https://github.com/Telefonica/mistica-design/discussions/1196
https://github.com/Telefonica/mistica-design/discussions/1198

### Do you have a visual proposal?

![Comp C](https://user-images.githubusercontent.com/129896641/233357428-c813461c-0b0c-46eb-942d-cdac7abba027.jpg)

**Comment by yceballost** [comment]
Buenas @vinrosa! 


Este componente pensamos lo mismo que el Slot A de la discussion https://github.com/Telefonica/mistica-design/discussions/1195,

Te copio lo mismo que hemos dicho en el Slota A

Hemos estado viendo la manera de encajar esto dentro de Mística como algo "Core" pero no lo vemos como algo que pueda ser core porque quizás está enfocado en cubrir casos de uso concretos.

Esto se consideraría un componente extended de momento.

Los pasos a seguir son:
- Hacéis specs desde vuestro lado. Estas specs son vuestra responsabilidad completamente y vivirán de momento en vuestro proyecto de Figma por considerarse un componente extended.
- El desarrollo es responsabilidad del equipo de Brasil.

# Desarrollo dentro de Mística
Estamos pendiente de confirmar que este slot necesita ser compartido entre 2 equipos de desarrollo. Si lo necesitan 2 equipos de desarrollo (novum + vivo) tendría que ir dentro del repositorio de Mística-web.

En el caso de que este componente terminara dentro de mistica-web se plantea de la siguiente manera:
- El componente no será visible en storybook
- El componente no tendrá documentación de diseño en Brand Factory
- El componente no estará disponible en la librería de Mística Mobile ni Mística Desktop en Figma, el componente estaría en vuestra librería intermediaria.
- El equipo de novum "bebería" del componente extended de la librería de Figma de Brasil
